plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -109,35 +113,42 @@ private static JSONObject generateHeteroDescribableConfigObject(HeteroDescribabl
             return finalHeteroConfiguratorObject;
     }
 
-    private static JSONObject generateNonEnumAttributeObject(Attribute attribute) {
+    private static JSONObject generateNonEnumAttributeObject(Attribute attribute, BaseConfigurator baseConfigurator) {
         JSONObject attributeType = new JSONObject();
         switch (attribute.type.getName()) {
             case "java.lang.String":
                 attributeType.put("type", "string");
+                attributeType.put("description", retrieveDocStringFromAttribute(baseConfigurator.getTarget(), attribute.name));
```

Comment:
could we just move  `retrieveDocStringFromAttribute(baseConfigurator.getTarget(), attribute.name)` above the switch case and extract a variable called `description` or `helpText`

Reply:
Yeah sounds like a plan :)